app/controllers/api/teams_controller.rb

```diff
@@ -28,6 +28,24 @@ def index
 
   private
 
+  def team_response
+    team = fetch_entries.first
```

Comment:
better @team ?

there's already a getter `def team`, is this still required?
maybe use this getter to cache team ?